graph/graph.go

```diff
@@ -31,6 +31,10 @@ type Graph interface {
 	// within the graph.
 	Has(id int64) bool
 
+	// Node returns the node with the given ID if it exists
+	// in the graph.
```

Comment:
```diff
 // Node returns the node with the given ID if it exists
-// in the graph.
+// in the graph, and nil otherwise.
```

Also update docs of concrete implementations.

Reply:
Done.